src/envoy/http/path_matcher/filter.cc

```diff
@@ -47,7 +48,7 @@ using RcDetails = Envoy::ConstSingleton<RcDetailsValues>;
 }  // namespace
 
 Envoy::Http::FilterHeadersStatus Filter::decodeHeaders(
-    Envoy::Http::RequestHeaderMap& headers, bool) {
+    RequestHeaderMap& headers, bool) {
```

Comment:
PR description says that path filterState is moved to path mather, but I cannot see any implementation changes to path_matcher/filter.cc.

Reply:
done.